pkgs/applications/audio/fluidsynth/default.nix

```diff
@@ -14,6 +14,11 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-7SuM7a8IIecAJ83QvJfUba/wArAEXywqL2HwVbeG8H4=";
   };
 
+  postPatch = ''
+    substituteInPlace CMakeLists.txt \
+      --replace "CMAKE_INSTALL_NAME_DIR" "_"
```

Comment:
What does this change? Also this needs to go to staging.

Reply:
Currently, fluidsynth fails to launch with

```
dyld[51633]: Library not loaded: /nix/store/q58wr28p636qmi87agcl790b2zfk4qs0-fluidsynth-2.3.0//nix/store/q58wr28p636qmi87agcl790b2zfk4qs0-fluidsynth-2.3.0/lib/libfluidsynth.3.dylib
  Referenced from: <no uuid> /nix/store/q58wr28p636qmi87agcl790b2zfk4qs0-fluidsynth-2.3.0/bin/fluidsynth
  Reason: tried: '/nix/store/q58wr28p636qmi87agcl790b2zfk4qs0-fluidsynth-2.3.0//nix/store/q58wr28p636qmi87agcl790b2zfk4qs0-fluidsynth-2.3.0/lib/libfluidsynth.3.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/nix/store/q58wr28p636qmi87agcl790b2zfk4qs0-fluidsynth-2.3.0//nix/store/q58wr28p636qmi87agcl790b2zfk4qs0-fluidsynth-2.3.0/lib/libfluidsynth.3.dylib' (no such file), '/nix/store/q58wr28p636qmi87agcl790b2zfk4qs0-fluidsynth-2.3.0//nix/store/q58wr28p636qmi87agcl790b2zfk4qs0-fluidsynth-2.3.0/lib/libfluidsynth.3.dylib' (no such file), '/usr/local/lib/libfluidsynth.3.dylib' (no such file), '/usr/lib/libfluidsynth.3.dylib' (no such file, not in dyld cache)
[1]    51633 abort      result/bin/fluidsynth
```

This is because in [CMakeLists.txt](https://github.com/FluidSynth/fluidsynth/blob/56dbf03a49ce981feaa5555f737da6a11c07d9c5/CMakeLists.txt#L366-L367), `CMAKE_INSTALL_NAME_DIR` is reset (to a wrong value) on Darwin. Replacing `CMAKE_INSTALL_NAME_DIR` with any dummy string fixes the build.

Reply:
Could you leave a (brief) comment indicating this? Otherwise someone is liable to remove it without realizing its importance.

Reply:
I updated the PR with a much better solution. Do I still need to add a comment on that?

Reply:
I would suggest so, because again, it's liable to get removed by someone thinking "it still builds, must be ok". Few people _run_ the application in question on all platforms when reviewing.

Reply:
Done.